lib/chef/provider/package/openbsd.rb

```diff
@@ -71,32 +89,45 @@ def remove_package(name, version)
         private
 
         def installed_version
-          if parts = @new_resource.package_name.match(/^(.+?)--(.+)/)
+          if parts = new_resource.package_name.match(/^(.+?)--(.+)/)
```

Comment:
Because this regex is used multiple places, I would like to see it turned into a constant or gotten from a method